### PR TITLE
[77] Database Design For A Transportation Company

### PR DESCRIPTION
**Pull request by MYKEL7**
## Task: `Database Design For A Transportation Company`
- `I created a database for a transportation company `

### CODE REVIEW

File: submissions/onyemeMichael/database/transportation.sql [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+-- Insert into trips table
+INSERT INTO trips (passenger_class, passenger_ticket, trip_fare, number_of_parents_children, number_of_siblings_spouses, point_of_embarkation, passenger_id)
+VALUES(1, 'DC43569', 70000.0000, 2, 0, 'Lagos', 1),
+      (2, 'DC76682', 50000.0000, 0, 1, 'Lagos', 2),
+      (1, 'DC36844', 70000.0000, 4, 1, 'Lagos', 3),
+      (2, 'DC56483', 50000.0000, 2, 1, 'Lagos', 4);
+
+
+-- Cabin classes Table
+CREATE TABLE cabin_classes (
```
Comment (adaugochi) [inline submissions/onyemeMichael/database/transportation.sql:57]: This table is not need

File: submissions/onyemeMichael/database/transportation.sql [hunk 2]
```diff
@@ -0,0 +1,87 @@
+CREATE DATABASE transportation;
+
+USE transportation;
+
+-- Classes Table
+CREATE TABLE classes (
```
Comment (adaugochi) [inline submissions/onyemeMichael/database/transportation.sql:6]: this table is not needed

File: submissions/onyemeMichael/database/transportation.sql [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+-- Insert into cabin_classes
+INSERT INTO cabin_classes(cabin_code, trip_id)
+VALUES('D65', 1),
+	  ('D54', 2),
+      ('D39', 3),
+      ('D42', 4);
+
+-- Accident Table
+CREATE TABLE accidents (
```
Comment (adaugochi) [inline submissions/onyemeMichael/database/transportation.sql:73]: This table requires just 3 columns

**Review by adaugochi — CHANGES_REQUESTED** [review]
Also, remove all insert queries from the tables

File: submissions/onyemeMichael/database/transportation.sql (hunk 3 above)
Comment (adaugochi) [reply submissions/onyemeMichael/database/transportation.sql:73]: Kindly read thoroughly the task you are given. This is not correct

**Review by adaugochi — CHANGES_REQUESTED** [review]
I ask you to remove those things I pointed, but you went ahead to comment them. Kindly remove them. They shouldn't be seen at all

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by MYKEL7** [comment]
Done.

**Review by adaugochi — APPROVED** [review]

**Pull request merged**